lib/release/notes/log.rb

```diff
@@ -67,7 +67,10 @@ def copy_single_tag_of_activity(tag_from:, tag_to: "HEAD")
       # @api private
       def find_last_tag_and_log
         last_tag = system_last_tag.delete!("\n")
-        return false unless system_log(tag_from: last_tag, label: all_labels).present?
+        unless system_log(tag_from: last_tag, label: all_labels).present?
```

Comment:
you can prob just make this more syntactical by using an `if...else`  and not returning false. Was rubocop yelling at you?